### PR TITLE
Currency list: Add Papua New Guinean kina

### LINKED ISSUE #18233: Currency list: Add Papua New Guinean Kina (PGK)
**Is your feature request related to a problem? Please describe.**
Allow Metabase display of the Papua New Guinean Kina (symbol K, code PGK)

**Describe the solution you'd like**
Add the Kina to the currency list

**How important is this feature to you?**
We are using Metabase for a system in use in Papua New Guinea, and some of the data involves amounts in PGK. This would be useful to make it clear that the currency used is the PGK.

### PR DESCRIPTION
**Pull request by raphaelmerx**
Closes https://github.com/metabase/metabase/issues/18233

Adds the Papua New Guinean kina (code PGK, symbol K) to the currency list.

### CODE REVIEW

File: shared/src/metabase/shared/util/currency.cljc [hunk 1]
```diff
@@ -634,6 +634,13 @@
           :rounding 0,
           :code "PEN",
           :name_plural "Peruvian nuevos soles"}],
+   [:PGK {:symbol "K",
+          :name "Papua New Guinean kina",
```
Comment (pawit-metabase) [inline shared/src/metabase/shared/util/currency.cljc:638]: Looks like we normally capitalize the singular name in other entries. Would it be appropriate to capitalize it here as well?

```suggestion
          :name "Papua New Guinean Kina",
```

Comment (raphaelmerx) [reply shared/src/metabase/shared/util/currency.cljc:638]: I usually see it capitalised, but Wikipedia showed it lowercase. Let's capitalise, thanks!

**Comment by paoliniluis** [comment]
I'm looping @noahmoss as I remember he migrated currencies from JS to Clojure land

**Comment by nemanjaglumac** [comment]
Adding @howonlee as well, because his knowledge of the world currencies is AMAZING :)

**Review by noahmoss — APPROVED** [review]
LGTM!

Just FYI, this change will allow PGK to show up in the currency list, but due to limitations in our current system for i18n, only formatting using the currency code (`PGK`) or name (`Papua New Guinean kina`) will be supported.

We don't actually use most of the metadata in this file. However we might in the future if we revise our approach to i18n. This general issue is tracked by #10785.

File: shared/src/metabase/shared/util/currency.cljc [hunk 2]
```diff
@@ -634,6 +634,13 @@
           :rounding 0,
           :code "PEN",
           :name_plural "Peruvian nuevos soles"}],
+   [:PGK {:symbol "K",
+          :name "Papua New Guinean Kina",
+          :symbol_native "K",
+          :decimal_digits 2,
+          :rounding 0,
```
Comment (howonlee) [inline shared/src/metabase/shared/util/currency.cljc:641]: there's no 1 or 2 toea anymore, to note - there's like 10 currencies where rounding in MB is 0 but rounding in real life is to the 5th or 10th subunit. I guess the justification is to chart trading and o&g and other stuff where people round to smaller decimals. we're just not consistent about this, I won't stop you

**Review by howonlee — APPROVED** [review]

**Comment by pawit-metabase** [comment]
Thank you for your contribution!

**Pull request merged**